app/src/main/java/com/kickstarter/libs/utils/ExperimentUtils.kt

```diff
@@ -20,6 +23,22 @@ object ExperimentUtils {
                 Pair("user_country", experimentData.user?.location()?.country() ?: Locale.getDefault().country)
         )
     }
+
+    fun checkoutTags(experimentRevenueData: ExperimentRevenueData): MutableMap<String, Any?> {
+        val amount = experimentRevenueData.checkoutData.amount()
+        val project = experimentRevenueData.pledgeData.projectData().project()
+        val fxRate = project.fxRate()
+        val paymentType = experimentRevenueData.checkoutData.paymentType()
+        val revenue = (amount * fxRate * 100).roundToInt()
+        return mutableMapOf(
```

Comment:
Would it be better to return a regular `Map` here, if we don't mutate this data?

Reply:
Yeah, that makes sense! And I can do the same for `attributes`